precompiles/utils/src/testing/execution.rs

```diff
@@ -222,6 +222,32 @@ impl<'p, P: PrecompileSet> PrecompilesTester<'p, P> {
 		self.assert_optionals();
 	}
 
+	/// Execute the precompile set and check if it reverts.
+	/// Take a closure allowing to perform custom matching on the output.
+	/// Output is not automatically decoded.
+	pub fn execute_reverts_no_decode(mut self, check: impl Fn(&[u8]) -> bool) {
```

Comment:
@nanocryk I don't really want to add this, I just did to make it work for now.

Why does `execute_reverts()` automatically attempt to decode the returned output? This was causing a panic in my case because `b"GMP Precompile is not enabled"` fails to decode.